README_CN.md

```diff
@@ -0,0 +1,621 @@
+
+此扩展的操作指南可参考[乐鑫文档](https://docs.espressif.com/projects/vscode-esp-idf-extension/en/latest/)或 [Github 文档](./docs/ONBOARDING.md)。
+
+# 使用指南
+
+## 安装
+
+1. 下载并安装 [Visual Studio Code](https://code.visualstudio.com/)。
+
+2. 在操作系统中安装 ESP-IDF 所需的软件包：
+
+- 适用于 [MacOS 和 Linux](https://docs.espressif.com/projects/esp-idf/zh_CN/latest/esp32/get-started/macos-setup.html#install-prerequisites) 的软件包。
+- Windows 系统无需额外安装软件包。
+
+3. 打开 VS Code，点击左侧活动栏中的扩展图标，或使用**查看：显示扩展**命令（快捷键：<kbd>⇧</kbd><kbd>⌘</kbd><kbd>X</kbd> 或 <kbd>Ctrl+Shift+X</kbd>）。
+
+4. 搜索 [ESP-IDF 扩展](https://marketplace.visualstudio.com/items?itemName=espressif.esp-idf-extension)。
+
+5. 安装上述扩展。安装成功后，![Espressif 图标](./media/readme/espressifIcon.png) 会出现在 VS Code 左侧活动栏中。点击该图标，可以看到该扩展提供的基本命令列表。
+
+<p>
+  <img src="./media/readme/commandsList.png" alt="命令列表">
+</p>
+
+6. 从命令列表中选择 **配置 ESP-IDF 扩展** 或按 <kbd>F1</kbd> 输入 `Configure ESP-IDF Extension`，然后选择 **ESP-IDF：配置 ESP-IDF 扩展** 选项。
+   > **注意：** 对于 ESP-IDF 5.0 之前的版本，配置路径中不可出现空格。
+
+<p>
+  <img src="./media/readme/setup.png" alt="选择 ESP-IDF" width="950">
+</p>
+
+7. 选择 **Express** 后可自行切换下载服务器：
+
+- Espressif：该服务器链接在中国的下载速度更快。
+- Github：使用 Github 发布链接。
+
+8. 选择要下载的 ESP-IDF 版本，或选择 `Find ESP-IDF in your system` 选项查找系统中已有的 ESP-IDF 目录。
+
+9. 选择 ESP-IDF 工具的存放位置（即 `IDF_TOOLS_PATH`），默认情况下在 MacOS/Linux 系统中是 `$HOME\.espressif`，Windows 系统中是 `%USERPROFILE%\.espressif`。
+
+10. 如果使用 MacOS/Linux 操作系统，请选择系统 Python 可执行文件来在 ESP-IDF 工具内创建 ESP-IDF 虚拟环境，并安装 ESP-IDF Python 包。
+
+    > **注意：** Windows 用户不需要选择 Python 可执行文件，因为此设置过程会自动安装所需文件。
+
+11. 确保 `IDF_TOOLS_PATH` 中不包含空格，避免构建过程中出现问题，且 `IDF_TOOLS_PATH` 与 `IDF_PATH` 不能相同。
+
+12. 此时应出现安装界面，显示设置进度状态，包括 ESP-IDF 下载进度、ESP-IDF 工具的下载和安装进度，以及 Python 虚拟环境的创建过程。
+
+13. 如果一切正常，将收到所有设置已配置完成的消息，此时可开始使用扩展。
+
+如有问题，请参阅[故障排除](#Troubleshooting)部分。
+
+##  在 VSCode 中使用 ESP-IDF 扩展
+
+ESP-IDF 扩展在 VS Code 底部蓝色窗口的状态栏中提供了一系列命令图标，将鼠标悬停在图标上时，会看到可执行的命令。
+
+<p>
+  <img src="./media/readme/statusBar.png" alt="状态栏">
+</p>
+
+以下步骤展示了这些图标的常见用例：
+
+1. 按 <kbd>F1</kbd> 并输入 **ESP-IDF：展示示例项目**，可以基于 ESP-IDF 示例创建新项目。在命令面板中选择 ESP-IDF，搜索想使用的示例并创建新项目。
+
+2. 创建好新项目并在 VS Code 中打开后，点击状态栏图标 ![串口](./media/readme/serialport.png) 设置设备的串口。也可以按 <kbd>F1</kbd> 输入 **ESP-IDF：选择要使用的端口**，选择设备连接的串口。
+
+3. 点击状态栏图标 ![IDF 目标](./media/readme/target.png) 选择使用的芯片设备（如 esp32、esp32s2 等），或按 <kbd>F1</kbd> 输入 **ESP-IDF：设置乐鑫设备目标** 命令。
+
+4. 接下来，通过点击状态栏图标 ![sdkconfig 编辑器](./media/readme/sdkconfig.png) 或按 <kbd>F1</kbd> 输入 **ESP-IDF：SDK 配置编辑器** 命令（快捷键：<kbd>CTRL</kbd> <kbd>E</kbd> <kbd>G</kbd>），修改 ESP-IDF 项目设置。完成所有更改后，点击 `Save` 并关闭此窗口。可以在菜单栏中的`查看` -> `输出`中选择下拉列表里的 `ESP-IDF` 来查看输出信息。
+
+5. （可选）**ESP-IDF：运行 idf.py reconfigure 任务** 命令生成 `compile_commands.json` 文件，以便启用语言支持。也可以按照 [C/C++ 配置](./docs/C_CPP_CONFIGURATION.md) 文档中的说明配置 `.vscode/c_cpp_properties.json`。
```

Comment:
Hi @April-Yjj. Actually, ``idf.py reconfigure`` is the full name of the command. See more information [here](https://docs.espressif.com/projects/esp-idf/en/stable/esp32/api-guides/tools/idf-py.html#reconfigure-the-project-reconfigure). So it might be better to keep it as is. Thx~